crates/bevy_ecs/src/event.rs

```diff
@@ -164,13 +168,23 @@ pub struct EventWriter<'w, 's, T: Resource> {
 }
 
 impl<'w, 's, T: Resource> EventWriter<'w, 's, T> {
+    /// Sends an `event`. [`EventReader`]s can then read the event.
+    /// See [`Events`] for details.
     pub fn send(&mut self, event: T) {
         self.events.send(event);
     }
 
     pub fn send_batch(&mut self, events: impl Iterator<Item = T>) {
         self.events.extend(events);
     }
+
+    /// Sends the default value of the event. Useful when the event is an empty struct.
+    pub fn fire(&mut self)
```

Comment:
I know `fire` is snappier, but in a way `fire` is just a synonym for `send`. I personally think this should be called `send_default`. It is more typing, but also more descriptive because it builds on existing verbage instead of inventing new verbage.

Reply:
Rebased and renamed